### PR TITLE
Object Storage v1: Adding Timestamp to GetHeader results

### PR DESCRIPTION
**Pull request by jtopjian**
For #2170 

https://github.com/openstack/swift/blob/b79ca57fa98323b17f896af86c617d465066d544/swift/common/swob.py#L1014-L1039

### CODE REVIEW

File: openstack/objectstorage/v1/containers/results.go [hunk 1]
```diff
@@ -104,16 +105,19 @@ type GetHeader struct {
 	StoragePolicy    string    `json:"X-Storage-Policy"`
 	TempURLKey       string    `json:"X-Container-Meta-Temp-URL-Key"`
 	TempURLKey2      string    `json:"X-Container-Meta-Temp-URL-Key-2"`
+	Timestamp        float64   `json:"-"`
```
Comment (kayrus) [inline openstack/objectstorage/v1/containers/results.go:108]: @jtopjian you can use `ContentLength  int64     `json:"Content-Length,string"``, see for example https://github.com/gophercloud/gophercloud/blob/c9471fccb2bb90d41832195b1134ac1c6f50f2bb/openstack/objectstorage/v1/objects/results.go#L136

Comment (jtopjian) [reply openstack/objectstorage/v1/containers/results.go:108]: Oh nice - thanks!

**Comment by jtopjian** [comment]
recheck

**Comment by jtopjian** [comment]
The Container tests pass on Vexxhost. I'm thinking the OpenLab failure was a timing issue.

**Pull request merged**